src/Images.jsx

```diff
@@ -161,26 +153,22 @@ class Images extends React.Component {
                 onClick={() => this.deleteImage(image)}
             />
         ];
-        return <Listing.ListingRow
-                    key={image.id}
-                    rowId={image.id}
-                    columns={columns}
-                    tabRenderers={tabs}
-                    navigateToItem={this.navigateToImage(image)}
-                    listingActions={actions}
-        />;
+        return (
+            <React.Fragment key={image.id}>
+                <Listing.ListingRow
+                        rowId={image.id}
+                        columns={columns}
+                        tabRenderers={tabs}
+                        navigateToItem={this.navigateToImage(image)}
+                        listingActions={actions} />
+            </React.Fragment>
+        );
```

Comment:
What does this do? It just wraps a single element, as before?

Reply:
My fault. Removed and added the key property to the containing element.